apps/ui/src/views/Space/Overview.vue

```diff
@@ -38,7 +38,7 @@ watchEffect(() => setTitle(props.space.name));
       </div>
       <div class="relative bg-skin-bg h-[16px] top-[-16px] rounded-t-[16px] md:hidden" />
       <div class="absolute right-4 top-4 space-x-2">
-        <router-link :to="{ name: 'editor' }">
+        <router-link :to="{ name: 'editor' }" tabindex="-1">
```

Comment:
Why those changes (also in Proposals.vue)? I can't see a difference in behaviour.

Reply:
Those tabindex changes are only visible when using the keyboard tab key to navigate.

without the changes, button inside a <a> element are also focusable, making the button being focusable twice

Reply:
Was able to reproduce it now, thanks.